core/src/main/scripts/importer/validateData.py

```diff
@@ -1227,6 +1230,8 @@ def checkLine(self, data):
                         'No Amino_Acid_Change or HGVSp_Short value. This '
                             'mutation record will get a generic "MUTATED" flag',
                         extra={'line_number': self.line_number})
+        if ("NCBI_Build") not in self.cols or data[self.cols.index('NCBI_Build')] not in ["GRCh37", "GRCh38","GRCm38"]:
```

Comment:
I think we already have a validation for this field. I actually meant a warning for the **meta** file property that is read in the java code in `GeneticProfileReader.java` :  `String genomeBuild = properties.getProperty("reference_genome_id");`

Reply:
I see latest code uses the correct method :+1: